anydex/wallet/ethereum/eth_wallet.py

```diff
@@ -180,45 +158,37 @@ def get_transactions(self):
 
         return succeed(transactions_to_return)
 
-    def _update_database(self, transactions):
-        """
-        Update transactions in the database.
-        Pending transactions that have been confirmed will be updated to have a block number and will no longer be
-        pending.
-        Other transactions that are not in the database will be added.
-
-        :param transactions: list of transactions retrieved by self.provider
-        """
-        pending_transactions = self._session.query(Transaction).filter(Transaction.is_pending.is_(True)).all()
-        confirmed_transactions = self._session.query(Transaction).filter(Transaction.is_pending.is_(False)).all()
-        self._logger.debug('Updating ethereum database')
-        for transaction in transactions:
-            if transaction in pending_transactions:
-                # update transaction set is_pending = false where hash = ''
-                self._session.query(Transaction).filter(Transaction.hash == transaction.hash).update({
-                    Transaction.is_pending: False,
-                    Transaction.block_number: transaction.block_number
-                })
-            elif transaction not in confirmed_transactions:
-                self._session.add(transaction)
-        self._session.commit()
-
-    def get_transaction_count(self):
-        """
-        Get the amount of transactions sent by this wallet
-        """
-        row = self._session.query(Transaction.nonce).filter(Transaction.from_ == self.get_address().result()).order_by(
-            Transaction.nonce.desc()).first()
-        if row:
-            return row[0] + 1  # nonce + 1
-        return 0
-
     def min_unit(self):
```

Comment:
If the minimum unit is 1, shouldn't the precision also be 1?

Reply:
dont think so, I believe precision is how much 1 ethereum can be subdevided while min unit is the minimum that can be traded.